src/test/java/org/jabref/model/groups/AutomaticPersonsGroupTest.java

```diff
@@ -0,0 +1,59 @@
+package org.jabref.model.groups;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.strings.LatexToUnicodeAdapter;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AutomaticPersonsGroupTest {
+    private static Set<GroupTreeNode> createPersonSubGroupFrom(String... lastNames) {
+        return Arrays.stream(lastNames)
+                     .distinct()
```

Comment:
please remove distinct and the latextounicode part here. The tests should be a as stupid as possible, e.g. if you want to test that the code transforms latex to unicode, you better do it by hand.

Reply:
This is what I use to generated the expected result.
1. I am using latextounicode part here because I use the last name both as group name and search expression. As it is the expected subgroup they must be the same. (and without latex)
2. The distinct is implied from the set. Given that this is used to generate the expected result, if something messes up in the equality/hashing of the group, it is likelier to be caught by using distinct Strings to create the groups.

That said, if you don't agree I'll change both.

Reply:
I think `createPersonSubGroupFrom(`A`, `A`)` should create two groups. It's the responsible of the caller to make sure she passes distinct values. Similar, one should pass `Römer` instead of `R\"omer` if one expect that latex is resolved.
The current test passes if latex2unicode is changed to return always the same string that was put into it. That's not desired.

Reply:
Unless I have missed something, the tests should now be as stupid as they can be ^^